modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/BaseServerComponent.java

```diff
@@ -143,6 +144,12 @@ public AvailabilityType getAvailability() {
                 previousAvailabilityType = DOWN;
                 availabilityType = DOWN;
                 securityRealmReady = false;
+            } catch (UnauthorizedException e) {
+                previousAvailabilityType = DOWN;
+                availabilityType = DOWN;
+                throw new InvalidPluginConfigurationException("User authenticated via HTTP management interface does"
```

Comment:
>
> Mmm, I don't think it's needed. If ASConnection#execute throws IPCE
> because of invalid credentials then we should just let the exception
> bubble up.

It won't bubble up, because it's caught by  
https://github.com/lzoubek/rhq/blob/bugs/1220277/modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/BaseServerComponent.java#L169  
Then resource error is not stored and user does not know why's the  
resource down (this should be separate BZ)